packages/language-server/src/__test__/completion.test.ts

```diff
@@ -1614,6 +1615,36 @@ suite('Completions', function () {
         },
       })
     })
+    test('MongoDB: @@index([email, address.alpha.bravo.hello|]) with composite type suggestion, depth 3', () => {
+      assertCompletion({
+        provider: 'mongodb',
+        schema: /* Prisma */ `
+          type Address {
+            street String
+            number Int
+            alpha  Alpha
+          }
+          type Alpha {
+            bravo  Bravo
+            helloA Int
+          }
+          type Bravo {
+            something  String
+            helloBravo Int
+          }
+          model User {
+            id      Int     @id @map("_id")
+            email   String
+            address Address
+            @@index([email, address.alpha.bravo.hello||])
```

Comment:
The `||` is a bug here I guess?

Reply:
Nice 😄 the code still works, but I'll fix that